### DISCUSSION TITLE
Repositório com pendências, falhas de operação e problemas de privacidade do Pix

### DISCUSSION THREAD

**Discussion by renatofrota** [bacen/pix-api#200] [Show and tell]
Não vejo sentido em ficarmos abrindo reclamações/denúncias no Sistema de Registro de Demandas do Cidadão (RDR) do Banco Central para os PSPs (Prestadores de Serviços de Pagamentos) individualmente, devido o grande fluxo de registros repetidos (que eu suponho que estejam sendo enviados) para um mesmo problema, que acabam atrasando a resolução em vez de agilizar - e o que mais queremos neste momento é a célere adequação de todos os PSPs aos regulamentos do Pix, aderindo ao padrão e garantindo a perfeita interoperabilidade entre todos.

Por isso, resolvi criar um repositório com uma lista de pendências e falhas de implementação da API e de outros procedimentos relativos a operação do Pix. Assim podemos concentrar num só local o que não está funcionando adequadamente e marcar os PSPs que estão pendentes a efetuar cada adequação.

https://github.com/renatofrota/pix-pendencias

Convido os interessados a acompanhar.

**Comment by rodrigonunes100** [comment]
Notícia : BC publica resolução isentando empresas de multas até maio de 2021
https://www.contabeis.com.br/noticias/45231/pix-bc-publica-resolucao-isentando-empresas-de-multas-ate-maio-de-2021/

**Comment by renatofrota** [comment]
> Notícia : BC publica resolução isentando empresas de multas até maio de 2021
> https://www.contabeis.com.br/noticias/45231/pix-bc-publica-resolucao-isentando-empresas-de-multas-ate-maio-de-2021/

É um prazo grande, mas essa atitude já era esperada. O calendário me pareceu apertado. Acompanho como mero observador há pouco mais de 1 mês esse repositório aqui e estou vendo a correria que está sendo pra todos.

**Comment by renatofrota** [comment]
Panorama Geral de Operação e Interoperabilidade Pix:
https://github.com/renatofrota/pix-pendencias/issues/22
